autoswap/config.go

```diff
@@ -58,30 +66,22 @@ func (b Balance) String() string {
 	return b.Relative.String()
 }
 
+type SerializedConfig = autoswaprpc.Config
+
 type Config struct {
-	Enabled             bool
-	ChannelPollInterval uint64
-	LiquidAddress       string
-	BitcoinAddress      string
-	MaxBalance          uint64
-	MinBalance          uint64
-	MaxBalancePercent   utils.Percentage
-	MinBalancePercent   utils.Percentage
-	MaxFeePercent       utils.Percentage
-	AcceptZeroConf      bool
-	FailureBackoff      uint64
-	Budget              uint64
-	BudgetInterval      uint64
-	Currency            boltz.Currency
-	Type                boltz.SwapType
-	PerChannel          bool
-	Wallet              string
-	MaxSwapAmount       uint64
-
-	maxBalance   Balance
-	minBalance   Balance
-	strategy     Strategy
-	strategyName string
+	*SerializedConfig
```

Comment:
Wouldn't it be smarter to have this as a function so that in case a value is changed, we can call a function and get the latest version of the serialized version?

Reply:
You mean like this?
https://github.com/BoltzExchange/boltz-client/pull/93/files/4fa62f7756b63ad5b27c627d825c08061b260597#r1517672007
Well need all the fields in the go version aswell in that case